packages/plugin-go/index.js

```diff
@@ -47,7 +48,11 @@ export default {
       return githubUrls(target);
     }
 
-    return [`https://${target}`, `https://golang.org/pkg/${target}`];
+    return [
+      `https://${target}`,
+      `https://golang.org/pkg/${target}`,
+      liveResolverQuery({ type: 'go', target }),
```

Comment:
I thought we're already using https://github.com/OctoLinker/live-resolver for `go` packages as a fallback 😄